### PR TITLE
[dashboard] Improve and fix workspace details on workspace list

### PR DESCRIPTION
**Pull request by svenefftinge**
- Show hover
- fix navigation issues

https://user-images.githubusercontent.com/372735/112756364-18208800-8fe5-11eb-9e96-ed641cdb6971.mov

### CODE REVIEW

**Comment by gtsiolis** [comment]
Looking at this now! 👀

File: components/dashboard/src/workspaces/WorkspaceEntry.tsx [hunk 1]
```diff
@@ -100,20 +101,23 @@ export function WorkspaceEntry({ desc, model }: { desc: WorkspaceInfo, model: Wo
         setChangesModalVisible(true);
     }
     return <div>
-        <a className="rounded-xl whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between hover:bg-gray-100 focus:bg-gitpod-kumquat-light cursor-pointer group" href={startUrl.toString()}>
+        <div className="rounded-xl whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between hover:bg-gray-100 focus:bg-gitpod-kumquat-light group">
             <div className="pr-3 self-center">
-                <div className={stateClassName}>
-                    &nbsp;
-            </div>
+                <Tooltip content={state}>
```
Comment (gtsiolis) [inline components/dashboard/src/workspaces/WorkspaceEntry.tsx:106]: **issue**: Looks great! The lowercase _stopping_ state is coming from the backend, right? Worth opening a follow up issue to update this to _Stopping_ if possible? WDYT?

Comment (gtsiolis) [inline components/dashboard/src/workspaces/WorkspaceEntry.tsx:106]: **suggestion**:  It's safe to remove the cursor pointer here. WDYT?

File: components/dashboard/src/workspaces/WorkspaceEntry.tsx [hunk 2]
```diff
@@ -100,20 +101,23 @@ export function WorkspaceEntry({ desc, model }: { desc: WorkspaceInfo, model: Wo
...
+                    <div className={stateClassName}>
+                    </div>
+                </Tooltip>
             </div>
             <div className="flex flex-col w-3/12">
-                <div className="font-medium text-gray-800 truncate">{ws.id}</div>
-                <a href={project ? 'https://' + project : undefined}><div className="text-sm overflow-ellipsis truncate text-gray-400">{project || 'Unknown'}</div></a>
+                <a href={startUrl.toString()}><div className="font-medium text-gray-800 truncate hover:text-blue-500">{ws.id}</div></a>
```
Comment (gtsiolis) [inline components/dashboard/src/workspaces/WorkspaceEntry.tsx:112]: **thought**: The idea was to have a rounded background on hover as in the pending changes column. Blue is not a color we've used for links elsewhere but let's get this is and fine tune within this week.

File: components/dashboard/src/components/Tooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ */
+
+import { useState } from 'react';
+
+export interface TooltipProps {
+    children: React.ReactChild[] | React.ReactChild;
+    content: string;
+}
+
+function Tooltip(props: TooltipProps) {
```
Comment (gtsiolis) [inline components/dashboard/src/components/Tooltip.tsx:14]: **praise**: Welcome, tooltip component! 🎈

**Review by gtsiolis — APPROVED** [review]
[Smooth](https://youtu.be/h_D3VFfhvs4)!

File: components/dashboard/src/workspaces/WorkspaceEntry.tsx (hunk 1 above)
Comment (svenefftinge) [reply components/dashboard/src/workspaces/WorkspaceEntry.tsx:106]: Yeah, I can do a translation in this PR already.

File: components/dashboard/src/workspaces/WorkspaceEntry.tsx (hunk 2 above)
Comment (svenefftinge) [reply components/dashboard/src/workspaces/WorkspaceEntry.tsx]: We decided to go with blue-600.

File: components/dashboard/src/components/Tooltip.tsx (hunk 3 above)
Comment (svenefftinge) [reply components/dashboard/src/components/Tooltip.tsx:14]: 👋

**Pull request merged**